Contracts/v1.0/buyer/ad-details/response.json5

```diff
@@ -0,0 +1,66 @@
+//Response data
+{
+    "product":
+    {
+        //[Mandatory] id of the product
+        "id": "fad5b5",
+        //[Mandatory] title of product
+        "title": "I-Phone 11s",
+
+        //[Mandatory] description of product
+        "description": "lorem lorem lorem lorem lorem lorem lorem lorem lorem ",
+
+        //[Mandatory] images of the product
+        "heroImageUrl": "www.images.com/imageName" ,
+
+        //[Mandatory] price of product
+        "price": 
+        {
+            "amount":"90000",
+            "isPriceNegotiable":"true"
+        },
+        //[Mandatory] category of product
+        "category": "Accessories",
+
+        /[Mandatory] status of the product i.e. Active/Disable/Sold
+        "status" : "Active",
+
+        //[Mandatory] date when product ad was posted in yyyy'-'MM'-'dd format
+         "postDate" : "2019-09-01",
+
+        //[Mandatory] date when the ad will expire in yyyy'-'MM'-'dd format
+        "expirationDate" : "2019-10-01",
+
+         //[Optional] list of images
+        "imageUrls" : [
+        {
+            "url" : "www.example.com/firstImage"
+        },
+        {
+            "url" : "www.example.com/secondImage"
+        }],
+      
+        //[Mandatory] description of seller
+        "seller": {
+            "id": "1111",
+            "name": "abees",
+            "image":{
+                "url" : "www.example.com/secondImage"
+            }
+        
+        },
+        //[Mandatory] date of purchase of the  product
+        "purchaseDate": "yyyy'-'MM'-'dd",
```

Comment:
What if the product is not for resale ?
Are we not considering those type of products .? Cause if we are then mandating purchasedate doesn’t make sense